### PR TITLE
refactor(frontend): logo inserted with better quality.

### LINKED ISSUE #2592: 🔧 [Refactor] Gradido Logo blurred
## 🔧 Refactor ticket

Gradido Logo is blurred due to reduced resolution. We should serve different Images for the different devices and sizes.

image
`Das Gradido-Logo mit Schrift ist unscharf. Können wir da eines mit höherer Auflösung nehmen?`

Hint: use `srcset` [see here](https://www.dofactory.com/html/img/sizes)

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
logo inserted with better quality. and the code made a little clean

### Issues
- fixes #2592 

Bildschirmfoto vom 2023-02-01 10-35-31
Bildschirmfoto vom 2023-02-01 10-34-27

### CODE REVIEW

**Comment by ogerly** [comment]
> Is the `green.png` still in use? If not, please delete

in in use

File: frontend/src/components/Auth/AuthNavbar.vue [hunk 1]
```diff
@@ -35,7 +35,8 @@ export default {
   mixins: [authLinks],
   data() {
     return {
-      logo: '/img/brand/green.png',
```
Comment (ulfgebhardt) [inline frontend/src/components/Auth/AuthNavbar.vue:38]: delete this file?

Comment (ogerly) [reply frontend/src/components/Auth/AuthNavbar.vue:38]: this must remain because of the exact dimensions

**Review by Mogge — COMMENTED** [review]
I still do not understand why `green.png` must be kept?

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**